docs/Skill_Tutorial.md

```diff
@@ -1,21 +1,21 @@
 # SUSI.AI Skill Development Tutorial
 
-Do you want your own AI Skills for SUSI? It's surprisingly easy to add more Skills to SUSI.
+Do you want your own AI Skills for SUSI? It's surprisingly easy to add more Skills to SUSI.AI
 
 ## Getting Started
 
-It's easy, DON'T PANIC. You don't need to be a software developer to enhance SUSI.
+It's easy, DON'T PANIC. You don't need to be a software developer to enhance SUSI.AI
 
 ### What you have to do
 
-We have a SUSI Skill CMS at https://skills.susi.ai. Here you can create public skills in a simpled editor window or private skill bots. Both can be created in the same way except that one is public and can be edited by anyone and the other is private, which can be edited only by you. The private skill bot also provides a Javascript embed code, which you can add to your website. You can configure colors and icons to suite your style.
+We have a SUSI Skill CMS at https://skills.susi.ai. Here you can create public SUSI.AI skills in a simpled editor window or private skill bots. Both can be created in the same way except that one is public and can be edited by anyone and the other is private, which can be edited only by you. The private skill bot also provides a Javascript embeded code, which you can add to your website. You can configure colors and icons to suite your style.
 
 ### Preparation to start testing
 
-To test the SUSI Skills you are editing, you can use the public chat at:
+To test the SUSI.AI Skills you are editing, you can use the public chat at:
 https://chat.susi.ai
 
-One you created a public skill you could also test it using the SUSI.AI Android Application at:
+OnCe you create a public SUSI.AI skill you could also test it using the SUSI.AI Android Application at:
```

Comment:
Once

Reply:
@fragm3 
Thanks for review and pointing the mistake.
I've made the required change.